braindecode/datasets/moabb.py

```diff
@@ -56,8 +56,13 @@ def _fetch_and_unpack_moabb_data(dataset, subject_ids=None, dataset_load_kwargs=
 
 
 def _annotations_from_moabb_stim_channel(raw, dataset):
-    # find events from stim channel
-    events = mne.find_events(raw)
+    # find events from the stim channel
+    stim_channels = mne.utils._get_stim_channel(None, raw.info, raise_error=False)
+    if len(stim_channels) > 0:
+        # returns an empty array if none found
+        events = mne.find_events(raw, shortest_event=0, verbose=False)
+    else:
+        events, _ = mne.events_from_annotations(raw, verbose=False)
```

Comment:
```suggestion
    events, _ = mne.events_from_annotations(raw, verbose=False)
```
I think we should always get the events using the annotations (with https://github.com/NeuroTechX/moabb/pull/696,  the annotations are always set in raw objects obtained from moabb)

Reply:
For some strange reason, it does not happen in all the datasets. I can try to run without this and fix this on the moabb side.